src/anupSJun20/AddIntegers.java

```diff
@@ -0,0 +1,22 @@
+package anupSJun20;
+/*WAP to sum all the numbers in given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Ouput: Sum of all numbers in String is: 53*/
+
+public class AddIntegers {
+	int sum = 0;
+	void findSumOfIntegers(String str) {
+		for(int index = 0;index<str.length();index++) {
+			if(Character.isDigit(str.charAt(index))) {
+				sum += Integer.parseInt(String.valueOf(str.charAt(index)));
```

Comment:
Here you should use Character class method to convert char to int.
Character.getNumericValue(ch).
Better optimized way.